Docs: Update details on the results page (#9193)
PostHog/posthog.com: contents/docs/experiments/testing-and-launching.mdx

Sometimes, in the beginning of an experiment, results can be skewed to one side while enough data is still being gathered. While peeking at preliminary results is not a problem, making quick decisions based on them is [problematic](/blog/ab-testing-mistakes#3-conducting-an-experiment-without-a-predetermined-duration). We advise you to wait until your experiment has had a chance to gather enough data.

### Advanced results metrics

- **Mean:** For trends, the number who completed the goal divided by the number who were exposed. 

- **Delta %:** The percentage change between the control and test variants.

- **Credible interval:** Represents a range within which we believe the true difference between the test variant and the control means lies, with 95% probability. In this context, the interval is expressed as a percentage change, indicating how much higher or lower the mean of the test variant could be compared to the control, based on the observed data and our prior beliefs.

- **Win probability:** The probability that variant is the best performing one.

## Ending an experiment

> When you're ready to end your experiments, you can click the **Make decision** button on the experiment page to roll out a variant and stop the experiment.

Reviewer: ```suggestion
When you're ready to end your experiments, you can click the **Ship a variant** button on the experiment page to roll out a variant and stop the experiment.
```

> ![Make decision](https://res.cloudinary.com/dmukukwp6/image/upload/make_decision_a46213b333.png)

Reviewer: ```suggestion
![Ship a variant](https://res.cloudinary.com/dmukukwp6/image/upload/Screenshot_2024_08_23_at_13_52_25_6c1cf85153.png)
```

> If you want more precise control over the releaseEvents, you can also set the release condition for the feature flag and stop the experiment manually.

Reviewer: ```suggestion
If you want more precise control over your release, you can also set the release condition for the feature flag and stop the experiment manually.
```
